templates/master/ml-model/index.js

```diff
@@ -0,0 +1,150 @@
+                        "Fn::GetAtt": [
+                            "QnABotSMEmbeddingModel",
+                            "ModelName"
+                        ]
+                    },
+                    "InitialVariantWeight": 1,
+                    "VariantName": "AllTraffic",
+                    "ServerlessConfig": {
+                        "MaxConcurrency" : 50,
+                        "MemorySizeInMB" : 4096                               
+                    }
+                }
+            ]
+        }
+    },
+    "QnABotSMProvisionedEmbeddingEndpoint": {
+        "Condition":"EmbeddingsSagemakerProvisioned",
+        "Type": "AWS::SageMaker::Endpoint",
+        "Properties": {
+            "EndpointConfigName": {
+                "Fn::GetAtt": [
+                    "QnABotSMProvisionedEmbeddingEndpointConfig",
+                    "EndpointConfigName"
+                ]
+            }
+        }
+    },
+    "QnABotSMServerlessEmbeddingEndpoint": {
+        "Condition":"EmbeddingsSagemakerServerless",
+        "Type": "AWS::SageMaker::Endpoint",
+        "Properties": {
+            "EndpointConfigName": {
+                "Fn::GetAtt": [
+                    "QnABotSMServerlessEmbeddingEndpointConfig",
+                    "EndpointConfigName"
+                ]
+            }
+        }
+    },
+    "QnABotSMEmbeddingModelExecutionRole": {
+        "Condition":"EmbeddingsSagemaker",
+        "Type": "AWS::IAM::Role",
+        "Properties": {
+            "AssumeRolePolicyDocument": {
+                "Statement": [
+                    {
+                        "Action": [
+                            "sts:AssumeRole"
+                        ],
+                        "Effect": "Allow",
+                        "Principal": {
+                            "Service": [
+                                "sagemaker.amazonaws.com"
+                            ]
+                        }
+                    }
+                ],
+                "Version": "2012-10-17"
+            },
+            "Path": "/",
+            "Policies": [
+                {
+                    "PolicyName": "S3Policy",
+                    "PolicyDocument": {
+                        "Version": "2012-10-17",
+                        "Statement": [
+                            {
+                                "Effect": "Allow",
+                                "Action": [
+                                    "s3:GetObject"
+                                 ],
+                                "Resource": [
+                                    {"Fn::Sub":"arn:aws:s3:::${BootstrapBucket}/${BootstrapPrefix}/ml_model/e5-large.tar.gz"}
+                                ]                      
+                            }
+                        ]
+                    }
+                }                
+            ],
+            "ManagedPolicyArns": [
+                "arn:aws:iam::aws:policy/AmazonSageMakerFullAccess"
```

Comment:
@rstrahan can we remove `AmazonSageMakerFullAccess`, and specify the required permissions only. This won't be approved by the Publisher/security review.

Reply:
Done. Will be merged in through a different PR.